classes/controller/AdminController.php

```diff
@@ -410,6 +410,9 @@ class AdminControllerCore extends Controller
 
     /** @var int Auth cookie lifetime */
     const AUTH_COOKIE_LIFETIME = 3600;
```

Comment:
A deprecation notice should be put above the constant and mention that the new property should be used instead